unsubscribe-automation/src/server.ts

```
@@ -0,0 +1,54 @@
+      return reply.status(400).send({ error: "Invalid URL provided" });
+    }
+    server.log.error(error);
+    return reply
+      .status(500)
+      .send({ error: "An error occurred during the unsubscribe process" });
+  }
+});
+
+const start = async () => {
+  try {
+    const port = process.env.PORT ? parseInt(process.env.PORT) : 5000;
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)